src/pages/_app.tsx

```diff
@@ -8,6 +8,10 @@ import { AppPropsWithLayout, DefaultLayout } from '@layouts';
 import { GlobalStyles, theme } from '@styles';
 import { ThemeProvider } from 'styled-components';
 
+import localFont from 'next/font/local';
+
+const pretendard = localFont({ src: '../../public/fonts/PretendardVariable.woff2' });
```

Comment:
```suggestion
const pretendard = localFont({ src: 'fonts/PretendardVariable.woff2' });
```
이렇게하면 안되나요?? 넥스트에서 public은 기본적으로 이렇게 될 텐데,,,